src/cli.ts

```diff
@@ -160,7 +162,6 @@ export function initArgs(argv: string[]): yargs.Argv<any> {
       coerce: parseJson,
       description:
         'override Electron BrowserWindow options (via JSON string); see https://github.com/nativefier/nativefier/blob/master/API.md#browserwindow-options',
-      type: 'string',
```

Comment:
Why the disappearance of this extra yargs validation? Better TS typing made them useless?

Reply:
No it's more that it was a lie. It's not a string (after coercion). It's an object.